README.md

```diff
@@ -742,6 +742,34 @@ You will find information on the repair operation in the `LOG` file inside the d
 
 Before calling `repair()`, close a database if it's using the same `location`.
 
+## Ordering
+
+All write operations are asynchronously dispatched on the node thread pool and therefore the ordering
+is not guaranteed.
```

Comment:
Nitpicking, but I prefer saying "not defined" because "not guaranteed" would imply that we do intend to have an order.

Side note, we don't have a max line length in our markdown (unlike Node.js). Skip the newlines.

```suggestion
Write operations don't have a defined order because they are asynchronously dispatched on the Node.js thread pool.
```